buildingspy/development/regressiontest.py

```diff
@@ -424,22 +424,21 @@ def _checkKey(self, key, fileName, counter):
         '''
 
         print("Opening {}".format(fileName)
-        filObj=open(fileName, mode="rt", encoding="utf-8")
-        filTex=filObj.readline()
-        # Strip white spaces so we can test strpos for zero.
-        # This test returns non-zero for partial classes.
-        filTex.strip()
-        strpos=filTex.find("within")
-        if strpos == 0:
```

Comment:
What you really want to use here is the `in` operator:
http://stackoverflow.com/a/3437070/874701